crates/service/src/tap/checks/value_check.rs

```diff
@@ -514,16 +518,26 @@ mod tests {
             "Should require minimal value"
         );
 
-        let signed_receipt =
-            create_signed_receipt(ALLOCATION_ID, u64::MAX, u64::MAX, 500000000000000).await;
+        let signed_receipt = create_signed_receipt(
+            SignedReceiptRequest::builder()
+                .value(500000000000000)
```

Comment:
You already assigned this magic number to a var `minimal_value`. Why not reuse that?